docs/consensus/registry.md

```diff
@@ -0,0 +1,155 @@
+# Registry
+
+The registry service is responsible for managing an entity/node/runtime public
+key and metadata registry.
+
+The service interface definition lives in [`go/registry/api`]. It defines the
+supported queries and transactions. For more information you can also check out
+the [consensus service API documentation].
+
+<!-- markdownlint-disable line-length -->
+[`go/registry/api`]: ../../go/registry/api
+[consensus service API documentation]: https://pkg.go.dev/github.com/oasislabs/oasis-core/go/registry/api?tab=doc
+<!-- markdownlint-enable line-length -->
+
+## Methods
+
+### Register Entity
+
+Entity registration enables a new entity to be created. A new register entity
```

Comment:
We should define somewhere (probably not here) what an "entity" is. This text should probably link to that definition.

Reply:
For reference, this is what we have in the operator docs: https://docs.oasis.dev/operators/architecture-overview.html#entities-and-key-management. It's a bit high-level/casual, matching the overall tone of those docs.

Reply:
Added.